mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -222,57 +223,63 @@ export const Columns = (props: ColumnsProps) => {
         };
 
         return (
-          <VerticalContainer>
+          <VerticalContainer className={styles.list}>
             <ol className={styles.numberedList}>
-              {row.original.jobsDone.map((jobDone) => (
+              {row.original.jobsDone.map((jobDone, index) => (
```

Comment:
could we simplify it?

Reply:
let's create task for refactoring please

Reply:
Task #371